lfu_nots.go

```diff
@@ -0,0 +1,214 @@
+package cache
+
+import "container/list"
+
+type LFUNoTS struct {
+	// list holds all items in a linked list
+	frequencyList *list.List
+
+	// holds the all cache values
+	cache Cache
+
+	// size holds the limit of the LFU cache
+	size int
+
+	currentSize int
+}
+
+type cacheItem struct {
+	// key of cache value
+	k string
```

Comment:
we hold the key-value pair together with its usage count inside freqElement in this struct. 
It looks like we need. Dont we ?